app/models/admin.rb

```diff
@@ -1,4 +1,11 @@
 class Admin < ApplicationRecord
   devise :database_authenticatable, :lockable, :recoverable, :rememberable,
          :timeoutable, :trackable, :validatable
+
+  enum role: [
+    :owner,
```

Comment:
The name owner is ambiguous.

Reply:
We both agree, but are unsure what to call it. We'll keep it as `:owner` for now and change in the future.